src/Network/Network/help/New-AzNetworkManagerSecurityUserRule.md

```diff
@@ -0,0 +1,394 @@
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Default
+```
+New-AzNetworkManagerSecurityUserRule -Name <String> -RuleCollectionName <String>
+ -SecurityUserConfigurationName <String> -NetworkManagerName <String> -ResourceGroupName <String>
+ -Flag <String> [-Force] [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzNetworkManagerSecurityUserRule** cmdlet creates a security user rule.
+
+## EXAMPLES
+
+### Example 1: Create Custom Security User Rule
+```powershell
+$sourceAddressPrefix = New-AzNetworkManagerAddressPrefixItem -AddressPrefix "Internet" -AddressPrefixType "ServiceTag"
+$destinationAddressPrefix = New-AzNetworkManagerAddressPrefixItem -AddressPrefix "10.0.0.1" -AddressPrefixType "IPPrefix" 
+$sourcePortList = @("100")
+$destinationPortList = @("99")
+New-AzNetworkManagerSecurityUserRule -ResourceGroupName "psResourceGroup" -NetworkManagerName "psNetworkManager" -ConfigName "psSecurityUserConfig" -RuleCollectionName "psRuleCollection" -Name "psRule" -Description "TestDescription" -Protocol  "TCP" -Direction "Inbound" -SourcePortRange $sourcePortList -DestinationPortRange $destinationPortList -SourceAddressPrefix $sourceAddressPrefix -DestinationAddressPrefix $destinationAddressPrefix
+```
+
+```output
+Protocol                  : Tcp
+Direction                 : Inbound
+Sources                   : {Microsoft.Azure.Commands.Network.Models.NetworkManager.PSNetworkManagerAddressPrefixItem}
+Destinations              : {Microsoft.Azure.Commands.Network.Models.NetworkManager.PSNetworkManagerAddressPrefixItem}
+SourcePortRanges          : {100}
+DestinationPortRanges     : {99}
+SourcesText               : [
+                              {
+                                "AddressPrefix": "Internet",
+                                "AddressPrefixType": "ServiceTag"
+                              }
+                            ]
+DestinationsText          : [
+                              {
+                                "AddressPrefix": "10.0.0.1",
+                                "AddressPrefixType": "IPPrefix"
+                              }
+                            ]
+SourcePortRangesText      : [
+                              "100"
+                            ]
+DestinationPortRangesText : [
+                              "99"
+                            ]
+DisplayName               :
+Description               : TestDescription
+Type                      : Microsoft.Network/networkManagers/securityUserConfigurations/ruleCollections/rules
+ProvisioningState         : Succeeded
+SystemData                : Microsoft.Azure.Commands.Network.Models.NetworkManager.PSSystemData
+SystemDataText            : {
+                              "CreatedBy": "jaredgorthy@microsoft.com",
```

Comment:
can you remove my systemdata here?... any if you don't mind scrubbing other example files for it as well :)